### PR TITLE
Extend package signature validation options

### PR DESCRIPTION
**Pull request by vchaplinski**
Fixes #

## Proposed Changes

  - Support 3 options of package signature validation
    - Any signature
    - Trusted signature
    - Signature by whitelisted certificate
  - Removed option for certificates issues by whitelisted issuers
  - Added script to somewhat automate the localization routine

### CODE REVIEW

File: Up2dateService/Up2dateService/Service.cs [hunk 1]
```diff
@@ -24,19 +24,23 @@ protected override void OnStart(string[] args)
         {
             const int clientStartRertyPeriodMs = 30000;
 
-            System.Diagnostics.Debugger.Launch(); //todo: remove!
```
Comment (SteamDragon) [inline Up2dateService/Up2dateService/Service.cs:27]: Probably it's better to wrap into 
`#ifdef DEBUG
#endif`
?

Comment (vchaplinski) [reply Up2dateService/Up2dateService/Service.cs:27]: I don't want it pops up even in DEBUG mode. So I uncomment is just in rare cases when it is really needed.

Comment (SteamDragon) [reply Up2dateService/Up2dateService/Service.cs:27]: In that case comment is ok

File: Up2dateService/Tests/Up2dateTests/Up2dateShared/CertificateManagerTest.cs [hunk 2]
```diff
@@ -9,8 +9,6 @@ namespace Up2dateTests.Up2dateShared
     [TestClass]
     public class CertificateManagerTest
```
[the remark this answers is not in the document]
Comment (vchaplinski) [reply Up2dateService/Tests/Up2dateTests/Up2dateShared/CertificateManagerTest.cs:10]: Yes, but I don't want to delete it. Rework of CertificateManager is planned, so this unit tests placeholder will be filled in, I hope.

**Pull request merged**